apps/api/src/routers/user.py

```diff
@@ -63,6 +70,15 @@ async def apply(
     raw_application_data: Annotated[RawApplicationData, Depends(RawApplicationData)],
     resume: Optional[UploadFile] = None,
 ) -> str:
+    # Check if current datetime is past application deadline
+    now = datetime.now(timezone.utc)
```

Comment:
Issue: There's a `now` variable in this function and in the new function you created above. It might be simpler to add a time parameter to that new function and pass the now variable in so we don't have redundant variables.